### PR TITLE
Use named constant in stats aggregation output channel

### PR DESCRIPTION
**Pull request by wenleix**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java [hunk 1]
```diff
@@ -60,7 +60,7 @@
 {
     public static final int ROW_COUNT_CHANNEL = 0;
     public static final int FRAGMENT_CHANNEL = 1;
-    private static final int WRITER_CHANNELS = 2;
+    public static final int WRITER_CHANNELS = 2;
```
[the remark this answers is not in the document]
Comment (wenleix) [reply presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java]: @martint : These are the channels where aggregation stats starts: the first channel is `row_count` and the second channel is the metadata passed back from `PageSink`.

Start from the third channel, they are stats. So maybe call it `STATS_START_CHANNEL` ? 
@arhimondr , @electrum

[the remark this answers is not in the document]
Comment (arhimondr) [reply presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java]: I like that. Let's change it.

**Comment by wenleix** [comment]
Does the new constant name `STATS_START_CHANNEL` sounds good to you? @martint

**Comment by findepi** [comment]
I removed the `ready-to-merge` label because this is still "Changes requested". @martint please update your review.

**Pull request merged**